lib/matplotlib/pyplot.py

```diff
@@ -1408,6 +1408,12 @@ def axis(*v, **kwargs):
 
     The xmin, xmax, ymin, ymax tuple is returned
 
+      >>> axis('square')
```

Comment:
I think this is in the part of pyplot which is auto-generated by boilerplate.py so will be overwritten next time it is run.